main/reference/zoe-api/zoe.md

```diff
@@ -3,300 +3,333 @@
 // transform the untrusted invitation to a trusted one
 const invitation = await invitationIssuer.claim(untrustedInvitation);
 const { value: invitationValue } =
     await E(invitationIssuer).getAmountOf(invitation);
 ```
 
-## `E(zoe).getInvitationDetails(invitation)`
-- `invitation` `{Invitation}`
-- Returns `{Promise<Object>}`
+## E(Zoe).getInvitationDetails(invitation)
+- **invitation** **Invitation**
+- Returns **Promise&lt;Object>**
+
+Takes an **Invitation** as an argument and returns a **Promise** for an object containing the following
+details about the **Invitation**:
 
-Takes an `invitation` as an argument and returns an object containing the following
-details about the `invitation`:
-- `installation` `{Installation}`: The contract's installation in Zoe.
-- `instance` `{Instance}`: The contract instance this invitation is for.
-- `invitationHandle` `{Handle}`: A handle used to refer to this invitation.
-- `description` `{String}`: describes the purpose of this `invitation`. Use it
+- **installation** **Installation**: The contract's Zoe installation.
+- **instance** **[Instance](./zoe-data-types.md#instance)**: The contract instance this invitation is for.
+- **invitationHandle** **Handle**: A handle used to refer to this **Invitation**.
+- **description** **String**: Describes the purpose of this **Invitation**. Use it
    to match the invitation to the role it plays in the contract.
+
 ```js
 const invitation = await invitationIssuer.claim(untrustedInvitation);
-const invitationValue = await E(zoe).getInvitationDetails(invitation);
+const invitationValue = await E(Zoe).getInvitationDetails(invitation);
 ```
 
-## `E(zoe).install(bundle)`
-- `bundle` `{SourceBundle}`
-- Returns: `{Promise<Installation>}`
+## E(Zoe).install(bundle)
+- **bundle** **SourceBundle**
+- Returns: **Promise&lt;Installation>**
 
 Takes bundled source code for a Zoe contract as an argument and installs the code on Zoe.
-Returns an `installation` object.
-
-An `installation` is an object with one property:
-- `bundle`:  The contract source code, accessible via `bundle.source`, and other info.
+Returns a **Promise** for an **Installation** object.
 
 ```js
 // bundleSource takes source code files and
 // bundles them together in the format install expects.
-import bundleSource from '@endo/bundle-source';
+import bundleSource from '@endto/bundle-source';
 const bundle = await bundleSource(pathResolve(`./src/contract.js`));
-const installationP = await E(zoe).install(bundle);
+const installationP = await E(Zoe).install(bundle);
 ```
 
-## `E(zoe).getInstance(invitation)`
-- `invitation` `{Invitation}`
-- Returns: `{Promise<Instance>}`
+## E(Zoe).getConfiguration()
+- Returns: **Promise&lt;Object>**
+
+Returns the configuration settings for the Zoe contract.
+
+## E(Zoe).getFeeIssuer()
+- Returns: **Promise&lt;[Issuer](/reference/ertp-api/issuer.md)>**
+
+Returns a **Promise** for an **Issuer** that can mint ISTs.
```

Comment:
An `Issuer` can not "mint" anything. A `Mint` can. This needs to be clarified that the return is an `Issuer` for IST which can do stuff like create purses or payments.

Reply:
True, Issuers themselves can't mint anything. But all Issuers are associated with a Mint object (and a Brand object), and the associated Mint can mint stuff... such as ISTs, in this case. I could reword the description to explicitly state the chain of "Mint associated with the Issuer.... etc. etc.", but I don't think the chain is relevant here; explicitly mentioning the chain seems to be pointlessly verbose. I think it's understood that "Issuer that can mint ISTs" is shorthand for "Issuer associated with a Mint that can mint ISTs." If you disagree, I'll go ahead & update the text so the full chain is explicitly mentioned.

Reply:
I do disagree. I think this Issuer is not being made available because of its association with a Mint. I think it's available to make purses, deal with payments, etc. Issuers do a lot on their own without having a Mint nearby.

Reply:
Somewhere else we need to explain well the Brand/Issuer/Mint triple, which together represent one kind of asset --- like IST. Perhaps this is already explained well somewhere else that we can link to from here.

The Brand identifies the kind of asset. The Issuer facilitates exclusive rights transfer of those assets --- hence its concern with Payments and Purses. The Mint is indeed where new units of that asset come from.

Reply:
K... then I've changed the description to "Returns a Promise for an Issuer whose associated Mint can mint IST."

As for describing the triplet in general, it's described on both the Issuer topic & the Mint topic... both of which are links from this description.